library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/datasketches/cardinality/binaryoperator/HllUnionAggregator.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Is it worth adding another walkthrough for the doc module for HllpUnion, similar to UnionSketch.java? Otherwise these serialisers and aggregators may just get lost and forgotten about.